manifests/e/eza-community/eza/0.20.14/eza-community.eza.locale.en-US.yaml

```
@@ -0,0 +1,49 @@
+# Created with WinGet Releaser v2 using Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: eza-community.eza
+PackageVersion: 0.20.14
+PackageLocale: en-US
+Publisher: eza community
+PublisherUrl: https://eza.rocks
+PublisherSupportUrl: https://github.com/eza-community/eza/issues
+PackageName: eza
+PackageUrl: https://github.com/eza-community/eza
+License: MIT

```

```suggestion
License: European Union Public License 1.2
```

Should the license also be modified?